.vscode/settings.json

```diff
@@ -6,7 +6,13 @@
 	"[powershell]": {
 		"editor.defaultFormatter": "ms-vscode.powershell"
 	},
-	"[javascript][typescript][json]": {
+	"[javascript]": {
+		"editor.defaultFormatter": "biomejs.biome"
+	},
+	"[typescript]": {
+		"editor.defaultFormatter": "biomejs.biome"
+	},
+	"[json]": {
```

Comment:
why tho? the grouping works tho

Reply:
It is, indeed, but I keep having issue with this. But you're right, I'll rever!